mod/thewire/actions/thewire/delete.php

```diff
@@ -10,7 +10,7 @@
 
 // Make sure we actually have permission to edit
 $thewire = get_entity($guid);
-if ($thewire->canEdit()) {
+if (!$thewire->canEdit()) {
```

Comment:
actually it should be `if (!$thewire->canDelete()) {`

Reply:
thanks for your help and guidance